packages/experiments/src/components/M365Nav/M365NavPage.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {
+  ExampleCard,
+  IComponentDemoPageProps,
+  ComponentPage,
+  PropertiesTableSet
+} from '@uifabric/example-app-base';
+import { M365NavExample } from './examples/M365Nav.Example';
+const M365NavExampleCode =
+  require('!raw-loader!experiments/src/components/M365Nav/examples/M365Nav.Example.tsx') as string;
+
+export class M365NavPage extends React.Component<IComponentDemoPageProps, {}> {
+  public render(): JSX.Element {
+    return (
+      <ComponentPage
+        title='M365 Nav'
+        componentName='M365NavToggler'
+        exampleCards={
+          <div>
+            <ExampleCard title='M365 Nav' isOptIn={ true } code={ M365NavExampleCode }>
+              <M365NavExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={ [
+              require<string>('!raw-loader!experiments/src/components/M365Nav/M365Nav.types.ts')
+            ] }
+            renderOnly={ ['IM365NavProps', 'IM365NavState'] }
+          />
+        }
+        overview={
+          <div />
+        }
+        bestPractices={
+          <div />
+        }
+        dos={
+          <div>
+            <ul>
+              <li>Use M365 Nav component in all M365 Admin Centers to acheive coherence</li>
```

Comment:
Instead of M365 language here, can we avoid anything product specific here? (The goal is we have a variant that anyone can use without calling themselves M365.)